.github/workflows/e2e.yaml

```diff
@@ -77,37 +77,19 @@ jobs:
           docker system prune --all -f
           echo "Disk usage after cleanup"
           sudo df -h
-      # FIXME: Workaround for Request Timeout issue of artifacts https://github.com/actions/download-artifact/issues/249
-      - name: "GitHub Actions: download build artifacts with retry"
+      - name: "Github Actions: build"
         if: steps.eval-params.outputs.run-type == 'github'
-        uses: Wandalen/wretry.action@a163f62ae554a8f3cbe27b23db15b60c0ae2e93c # master
-        with:
-          action: actions/download-artifact@v4
-          with: |
-            path: build
-            name: build-output
-          attempt_limit: 5
-          attempt_delay: 1000
-      - name: "GitHub Actions: extract artifacts"
+        run: |
+          make build
+      - name: "Github Actions: distributions"
         if: steps.eval-params.outputs.run-type == 'github'
         run: |
-          echo "Extracting artifacts..."
-          ARTIFACT=$(find build/distributions/out/*-linux-${{ env.E2E_PARAM_ARCH }}.tar.gz)
-          if [[ "$ARTIFACT" == "" ]]; then
-            echo "Could not find built artifact for linux(${{ env.E2E_PARAM_ARCH }})."
-            exit 1
-          fi
-          cat $ARTIFACT.sha256 | sha256sum --check
-
-          mkdir build/distributions/artifacts
-          tar -xzf $ARTIFACT -C build/distributions/artifacts
-
-          SRC_DIR=$(find build/distributions/artifacts/*/bin -type d)
-          DEST_DIR=build/artifacts-linux-${{ env.E2E_PARAM_ARCH }}
-          for BIN in $(ls $SRC_DIR); do
-            mkdir -p $DEST_DIR/$BIN
-            cp $SRC_DIR/$BIN $DEST_DIR/$BIN/
-          done
+          make -j build/distributions
+      - name: "Github Actions: images"
+        if: steps.eval-params.outputs.run-type == 'github'
+        run: |
+          make -j images
+          make -j docker/save
```

Comment:
Do we need `docker/save` here?

Reply:
Yes because of: https://github.com/kumahq/kuma/blob/5fba9cfe5e813dc940509b4dc3b75ed4d7f9e349/mk/e2e.new.mk#L15-L24 

and I wanted to keep changes to the minimum